palantirnet/federated-search-demo
# Solr 7

## JIRA Ticket(s)

https://palantir.atlassian.net/browse/OPENSOURCE-31

## Description

This branch attempts to get Solr 7.7.2 up and running.

## Testing instructions

1. Checkout this branch
1. Run `vagrant halt && vagrant destroy`
1. Run `vagrant box update`
1. Run `vagrant up`
1. Run `/opt/solr/bin/solr -v` and confirm it is `7.7.2`
1. `vagrant ssh` and run `phing install-all`
1. Run `phing solr-reindex-all` 
1. Go to http://d8.fs-demo.local/search-app and confirm 32 records.

---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)
